src/amp-story-player.js

```diff
@@ -22,6 +22,8 @@ import {
   parseUrlWithA,
   removeFragment,
 } from './url';
+// Source for this constant is css/amp-story-player-iframe.css
+import {cssText} from '../build/amp-story-player-iframe.css';
```

Comment:
Should we still just import this as the name `CSS`?  That's what we use in e.g. extension files

Reply:
`cssText` is the exported name when defined in `build-system/tasks/css.js`. Extension CSS goes through a different part of the build pipeline that names it `CSS` 🤷‍♀️